server/src/main/java/com/amos/server/EventGrabDemo.java

```diff
@@ -66,7 +68,7 @@ public boolean onTouch(View v, MotionEvent e) {
                     id = e.getPointerId(i);
                     x = e.getX(i);
                     y = e.getY(i);
-                    msg = String.format("Count %d Index %d", count, i);
+                    msg = "Test Count "+count+" Index "+i;
```

Comment:
why did you change this? String.format seems more elegant to me.

Reply:
probably to circumvent *Warning: Implicitly using the default locale is a common source of bugs*

Reply:
it's only a temporary demo class and i wanted to remove quickly the linting errors. So i think it's fine for this case ;)

Reply:
Use locale here too if you are using them in other places as well. Also use locale english,